common/src/main/java/org/iris_events/common/MessagingHeaders.java

```diff
@@ -24,6 +24,8 @@ public static class Message {
         public static final String SUBSCRIPTION_ID = "x-subscription-id";
         public static final String CORRELATION_ID = "x-correlation-id";
         public static final String CACHE_TTL = "x-cache-ttl";
+
+        public static final String APP_VERSION = "x-app-version";
```

Comment:
I would rename this to CLIENT_VERSION = "x-client-version"

Reply:
@tomazic89

Reply:
agreed